starter-code/src/sequencer.js

```diff
@@ -1,68 +1,135 @@
-var LetterSequence = function(){}
-
-LetterSequence.prototype.createSequence = function(sequence){
-  var characters      = sequence.split("");
-  var containerString = "";
-  var repeatCount     = 1;
-
-  for (var i = 0; i < characters.length; i++){
-    var currentChar = characters[i];
-    var prevChar    = characters[i - 1];
-    var nextChar    = characters[i + 1];
+class LetterSequence {
+  constructor (){
+  }
 
-    if (currentChar === prevChar){
-      repeatCount++
+  createSequence (sequence) {
+    let characters      = sequence.split("");
```

Comment:
You can use `const` instead of `let`.

Also, you can use the spread operator instead of `split`; like this: `const characters = [...sequence]`.